velox/core/PlanNode.h

```diff
@@ -532,6 +532,18 @@ class AggregationNode : public PlanNode {
       bool ignoreNullKeys,
       PlanNodePtr source);
 
+  AggregationNode(
+      const PlanNodeId& id,
+      Step step,
+      const std::vector<FieldAccessTypedExprPtr>& groupingKeys,
+      const std::vector<FieldAccessTypedExprPtr>& preGroupedKeys,
+      const std::vector<std::string>& aggregateNames,
+      const std::vector<Aggregate>& aggregates,
+      const std::vector<vector_size_t>& globalGroupingSets,
+      const std::optional<const std::string>& groupIdName,
```

Comment:
This needs to be FieldAccessTypedExprPtr for consistency with the rest of the code where input columns are specified using FieldAccessTypedExprPtr, not std::string.